README.md

```diff
@@ -23,5 +23,6 @@ Installing PiPot is pretty straightforward:
 `/usr/src/pipot` (so the respective locations would be `/usr/src/pipot/server`
 and `/usr/src/pipot/client`).
 * (Optional) `chmod +x /usr/src/pipot/server/install/install.sh`
+* (Optional, for image generation) `chmox +x /usr/src/pipot/client/bin/chroot.sh`
```

Comment:
There's a typo in there ;)

Also, I wouldn't make it optional.

Reply:
fix that :)